tools/coremark.py

```diff
@@ -0,0 +1,24 @@
+import subprocess
+import re
+import numpy
+
+iter = 10
+res = []
+for i in range(iter):
+    print("Running CoreMark benchmark - Run #{}".format(i + 1))
+    comp_proc = subprocess.check_output(
+        'build/rv32emu build/coremark.elf', shell=True).decode("utf-8") 
```

Comment:
You shall check the existence of `build/rv32emu` and validate its results for the first time.